PowerEditor/installer/nativeLang/german.xml

```diff
@@ -1263,7 +1282,7 @@ Fortsetzen?"/>
 			<NeedToRestartToLoadPlugins title="Notepad++ muss neu gestartet werden" message="Ein Neustart von Notepad++ zum Laden der Plugins ist erforderlich."/>
 			<SaveAllConfirm title="'Alles Speichern' Bestätigung" message="Sollen wirklich alle Dokumente gespeichert werden?
```

Comment:
@schnurlos
`<SaveAllConfirm title="..." />` was removed completely in english.xml

It is replaced with the `<DoSaveAll title="...">` which you have already added and translated above.

See: deab93f9b50f580bc276225755c2782b20cd255a